server/src/main/java/org/apache/iotdb/db/pipe/collector/realtime/PipeRealtimeDataRegionCollector.java

```diff
@@ -50,7 +50,7 @@ public void customize(PipeParameters parameters, PipeCollectorRuntimeConfigurati
 
     final PipeTaskCollectorRuntimeEnvironment environment =
         (PipeTaskCollectorRuntimeEnvironment) configuration.getRuntimeEnvironment();
-    dataRegionId = String.valueOf(environment.getRegionId());
+    dataRegionId = String.valueOf(environment.getRegionId().getId());
```

Comment:
Is String.valueOf unnecessary or not?

Reply:
Necessary, because DataRegion's reserved member variables are of type string